.swiftlint.yml

```diff
@@ -25,3 +25,4 @@ file_header:
                     \/\/  Created by .*? on \d{1,2}\/\d{1,2}\/\d{2}\.
                     \/\/  Copyright © \d{4} Realm\. All rights reserved\.
                     \/\/
+line_length: 120
```

Comment:
Maybe we could log a message if a configuration has line length explicitly set to 120, so that more people will be aware that this is SwiftLint's new default?